com.unity.multiplayer.mlapi/Runtime/Core/NetworkingManager.cs

```diff
@@ -185,6 +186,27 @@ internal set
         /// Gets if we are connected as a client
         /// </summary>
         public bool IsConnectedClient { get; internal set; }
+
+        private ClientObjMapNode<NetworkedClient, NetworkedObject> clientObjMapNode;
```

Comment:
I also suggest `_clientObjMapNode` naming convention for private members here too.

Reply:
this one is a little more subtle.  The "when in Rome" philosophy would say don't be the only place that differs from the rest of the code.  And in this case, the "Albin" coding standard does not have private variable prefixes.  I'm sticking with this, and if we want to re-factor the MLAPI existing code this would go along with that.